docs/src/main/sphinx/security/built-in-system-access-control.md

```diff
@@ -42,6 +42,9 @@ Trino offers the following built-in system access control implementations:
 * - `opa`
   - Use Open Policy Agent (OPA) for authorization. See
     [](/security/opa-access-control).
+* - `ranger`
+  - Apache Ranger policies are used to authorize access and apply dynamic
+    row-filters and column-masking. See [](/security/ranger-access-control).
```

Comment:
I would remove `and apply dynamic row-filters and column-masking`. Masking and filtering is not more special than any other authorization method the Ranger plugin supports so I see no reason why it has to be stressed here.

I skipped more docs for now.

Reply:
Done.